docs/user/metrics/custom_distribution.md

```diff
@@ -0,0 +1,86 @@
+# Custom Distribution
+
+Custom distributions are used to record the distribution of arbitrary values.
+
+It should be used only when direct control over how the histogram buckets are computed is required.
+Otherwise, look at the standard distribution metric types:
+
+* [Timing Distributions](timing_distribution.md))
+
+> Note: Custom distributions are currently only allowed for GeckoView metrics (the `gecko_datapoint` parameter is present).
+
+## Configuration
+
+Custom distributions have the following required parameters:
+
+  - `range_min`: (int) The minimum value of the first bucket
```

Comment:
```suggestion
  - `range_min`: (Number) The minimum value of the first bucket
```

Reply:
I think it's important to communicate that these are integers -- you can't set range 0-1, bucket_count of 100 and see 0.01, etc., for example.

Reply:
`Integers` then?

Reply:
Sounds good to me